.ci/install_crio.sh

```diff
@@ -53,5 +53,8 @@ fi
 echo "Install crio service (${crio_service_file})"
 sudo cp "${crio_service_file}" "${service_path}"
 
+echo "Reload systemd services"
+sudo systemctl daemon-reload
```

Comment:
You have to test if you're running on a system with systemd. Our current CI (Semaphore) uses Ubuntu 14.04, which means that we have upstart services.